rlw.sh

```diff
@@ -174,28 +175,29 @@ main () {
 		--title='Roblox Linux Wrapper '"$rlwversion"'-'"$branch"'' \
 		--window-icon="$RBXICON" \
 		--width=480 \
-		--height=230 \
-		--cancel-label='Quit' \
+		--height=256 \
+		--cancel-label='Exit' \
 		--list \
-		--text 'What option would you like?' \
+		--text 'What would you like to do on ROBLOX?' \
 		--radiolist \
 		--column '' \
 		--column 'Options' \
 		TRUE 'Play Roblox' \
-		FALSE 'Play Roblox (Legacy Mode)' \
-		FALSE 'Roblox Studio' \
+		FALSE 'Play Roblox (Basic Mode)' \
```

Comment:
I'm not sure the change from Legacy to Basic mode is appropriate. IIRC Legacy mode doesn't offer a way to log in to ROBLOX, so having to log in from either Firefox or Studio isn't very convenient on the first run.